software/github-ntfy.yml

```
@@ -0,0 +1,12 @@
+name: Github Ntfy
+website_url: https://github.com/BreizhHardware/ntfy_alerts
+description: Push notifications to NTFY when a new release is available on Docker Hub or Github.
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Python
```

```suggestion
platforms:
  - Docker
```

Docker is the only documented installation method

---

Yes I remove the python one Last Week 😅